reactioncommerce/reaction
# Fix for: Cloning products is not reactive

Resolves #3957
Impact: **major**  
Type: **bugfix**

## Issue
Cloning a product via clone button does apparently have no effect. Only refreshing the page shows that the product indeed has been cloned.

## Solution/Changes
Whenever a product is cloned an (unobserved) server side cursor in publications/collections/products.js changes its results. But that does
not result in the changes being pushed to the client, because server
publications are not reactive. Pullling in one of several  meteor packages
available was not an option, therefore I've decided to trick the client
to resubscribe whenever that happens.

## Breaking changes
none expected


## Testing
1. As a admin, create a product
2. Navigate back to product grid
3. Clone the product in the Grid settings via clone button
4. See the cloned product appearing immediately in product grid.
